src/main/java/games/strategy/triplea/delegate/BattleCalculator.java

```diff
@@ -1345,13 +1339,12 @@ public static int getUnitPowerForSorting(final Unit current, final boolean defen
       final Collection<TerritoryEffect> territoryEffects) {
     final boolean lhtrBombers = games.strategy.triplea.Properties.getLHTR_Heavy_Bombers(data);
     final UnitAttachment ua = UnitAttachment.get(current.getType());
-    int rolls;
+    final int rolls;
     if (defending) {
       rolls = ua.getDefenseRolls(current.getOwner());
     } else {
       rolls = ua.getAttackRolls(current.getOwner());
     }
```

Comment:
Sorry, i thought you had requested updates, it changes things a bit to comment on new scope and approve, versus comments that increase scope and also request that they be done. 

My points above do not quite really apply in this case. I do like seeing comments on nearby places that can be improved easily. But at the same time it is a bit dime a dozen, so there is a question of focus and also balance.